src/WebServer_DevicesPage.ino

```diff
@@ -395,6 +395,7 @@ void handle_devicess_ShowAllTasksTable(byte page)
     {
       LoadTaskSettings(x);
       struct EventStruct TempEvent;
+      byte spi_gpios[3] = {static_cast<byte>(-1)};
```

Comment:
This is probably not going to initialize the array to all 0xFF values.
The `= {0};` only works for values of 0.
Also maybe best to use the `int8_t` type instead of `byte`, which is an `uint8_t`.
If you compare to `-1`, I am not entirely sure what will happen here.

Reply:
Somewhere in my mind was stored that the GPIOs were all stored as byte in the settings, but I hadn't fact-checked that, and of course that isn't the case. Fixed.
Thanks for the heads-up 👍